Cargo.toml

```diff
@@ -17,9 +17,12 @@ lilv = "0.2"
 log = "0.4"
 lv2-sys = "2"
 lv2_raw = "0.2"
+ringbuf = "0.2.6"
```

Comment:
One last bit, can you bump the version of the livi crate from `"0.3.12"` to `"0.3.13"`?